k/keepalived/build_info.json

```diff
@@ -0,0 +1,20 @@
+{
+    "maintainer": "Vinod.K1@ibm.com",
+    "package_name": "keepalived",
+    "github_url": "https://github.com/acassen/keepalived",
+    "version": "v2.2.8",
+    "default_branch": "master",
+    "build_script": "keepalived_ubi_9.3.sh",
+    "package_dir": "k/keepalived",
+    "docker_cmd" : "docker build -t ${package_name}:$PACKAGE_VERSION ${dir}",
+    "docker_build": true,
+    "use_non_root_user": "false",
+    "validate_build_script": true,
+    "v*.*.*": {
+       "dir": "2.2.8_ubi9",
+       "build_script" : "keepalived_ubi_9.3.sh"
+    },
+    "2.0.4": {
+	      "build_script" : "keepalived_rhel_7.4.sh"
```

Comment:
@vinodk99 what should be dir folder here for dockerfile.
If we dont have better not add support for this in manage currency